packages/realm/src/app-services/App.ts

```diff
@@ -278,25 +279,34 @@ export class App<
       assert.function(fetch, "fetch");
     }
 
+    const syncClientConfig: binding.SyncClientConfig_Relaxed = {
+      multiplexSessions,
+      userAgentBindingInfo: App.userAgent,
+    };
+
     fs.ensureDirectoryForFile(fs.joinPaths(baseFilePath || fs.getDefaultDirectoryPath(), "mongodb-realm"));
     // TODO: This used getSharedApp in the legacy SDK, but it's failing AppTests
-    this.internal = binding.App.getApp(
-      binding.AppCacheMode.Disabled,
-      {
-        appId: id,
-        deviceInfo: App.deviceInfo,
-        transport: createNetworkTransport(fetch),
-        baseUrl,
-        defaultRequestTimeoutMs: timeout ? binding.Int64.numToInt(timeout) : undefined,
-      },
-      {
-        baseFilePath: baseFilePath ? baseFilePath : fs.getDefaultDirectoryPath(),
-        metadataMode: metadata ? toBindingMetadataMode(metadata.mode) : binding.MetadataMode.NoEncryption,
-        customEncryptionKey: metadata?.encryptionKey,
-        userAgentBindingInfo: App.userAgent,
+    this.internal = binding.App.getApp(binding.AppCacheMode.Disabled, {
+      appId: id,
+      deviceInfo: App.deviceInfo,
+      transport: createNetworkTransport(fetch),
+      baseUrl,
+      defaultRequestTimeoutMs: timeout ? binding.Int64.numToInt(timeout) : undefined,
+      baseFilePath: baseFilePath ? baseFilePath : fs.getDefaultDirectoryPath(),
+      metadataMode: metadata ? toBindingMetadataMode(metadata.mode) : binding.MetadataMode.NoEncryption,
+      customEncryptionKey: metadata?.encryptionKey,
+      syncClientConfig: {
         multiplexSessions,
-      },
-    );
+        userAgentBindingInfo: App.userAgent,
+      }
+    });
+  }
+
+  /**
+   * @internal
+   */
+  public get syncManager(): binding.SyncManager {
+    return this.internal.syncManager;
   }
```

Comment:
Don't see any references to this, when would this get used? In `Sync.ts` I see it used directly from the internal object, but not the SDK `App` object.

Reply:
Yeah, we don't need to expose it now